seqr/views/utils/file_utils.py

```diff
@@ -51,8 +55,8 @@ def parse_file(filename, stream):
         rows = rows[:last_row_index+1]
         # all rows should have same column count
         last_col_index = max(max(i for i, val in enumerate(row) if val) for row in rows)
-        padding = [None] * last_col_index
-        rows = [(row + padding)[:last_col_index+1] for row in rows]
+        padding = [''] * last_col_index
+        rows = [(row[:max(i for i, val in enumerate(row) if val)+1] + padding)[:last_col_index+1] for row in rows]
```

Comment:
`row + padding` will keep the trailing `None`. So, the `row` has to be replaced with `row[:max(i for i, val in enumerate(row) if val)+1]`

Reply:
hmm I see the issue. I think a neater way to handle this would be to change the return from `_parse_excel_string_cell` from `cell_value` to `cell_value or ''`

Reply:
Yes. Then, remove the padding? And replace all `None` value cells with `''`, not just the trailing ones?

Reply:
I keep the trailing padding for just in case there are excel files with shorter rows.